worker/src/RTC/DtlsTransport.cpp

```diff
@@ -1031,8 +1129,6 @@ namespace RTC
 		MS_TRACE();
 
 		MS_ASSERT(this->handshakeDone, "handshake not done yet");
-		MS_ASSERT(
```

Comment:
Here we should assert that `this->remoteFingerprint.algorithm.has_value()` right?

Reply:
Fingerprint.algorithm is not optional.